include/alpaka/workdiv/WorkDivUniformedCudaHipBuiltIn.hpp

```diff
@@ -105,86 +106,79 @@ namespace alpaka
         namespace traits
         {
             //#############################################################################
-            //! The GPU HIP accelerator work division grid block extent trait specialization.
+            //! The GPU CUDA-HIP accelerator work division grid block extent trait specialization.
             template<
                 typename TDim,
                 typename TIdx>
             struct GetWorkDiv<
-                WorkDivHipBuiltIn<TDim, TIdx>,
+                WorkDivUniformedCudaHipBuiltIn<TDim, TIdx>,
                 origin::Grid,
                 unit::Blocks>
             {
                 //-----------------------------------------------------------------------------
                 //! \return The number of blocks in each dimension of the grid.
-                ALPAKA_NO_HOST_ACC_WARNING
-#if defined(BOOST_COMP_HCC) && BOOST_COMP_HCC /* hcc requires matching host-device signature */
-                ALPAKA_FN_HOST_ACC
-#else /* nvcc does not know about blockDim.x etc. on host */
-                __device__
-#endif
-                static auto getWorkDiv(
-                    WorkDivHipBuiltIn<TDim, TIdx> const & workDiv)
+                __device__ static auto getWorkDiv(
+                    WorkDivUniformedCudaHipBuiltIn<TDim, TIdx> const & workDiv)
```

Comment:
Don't we need the workaround?

Reply:
Which workaround do you mean.

Reply:
The original code was:
```
ALPAKA_NO_HOST_ACC_WARNING
#if defined(BOOST_COMP_HCC) && BOOST_COMP_HCC /* hcc requires matching host-device signature */
    ALPAKA_FN_HOST_ACC
#else /* nvcc does not know about blockDim.x etc. on host */
    __device__
#endif
```

instead of only `__device__`